src/shogun/CMakeLists.txt

```diff
@@ -200,6 +200,8 @@ if (OPENMP_FOUND)
   SHOGUN_COMPILE_OPTS(${OpenMP_CXX_FLAGS})
   if (CMAKE_COMPILER_IS_GNUCC)
     SHOGUN_LINK_LIBS(gomp)
+  else ()
+    SHOGUN_LINK_LIBS(OpenMP::OpenMP_CXX)
```

Comment:
btw what happens if OpenML hasn't been found?

Reply:
you only reach this line if it has been found. this extends it to compilers other than gcc
https://github.com/shogun-toolbox/shogun/blob/f7255cf2cc6b5116e50840816d70d21e7cc039bb/src/shogun/CMakeLists.txt#L197-L206

Reply:
ah yes, sorry! missed that!